tests/test_helper.rb

```diff
@@ -0,0 +1,81 @@
+require 'minitest/autorun'
```

Comment:
Any reason why there's a separate testcases folder? Don't think these tests would be picked up with the automated test runs, the test command follows a pattern that picks up the files in spec folder with filename ending with `_spec.rb`. Anything else, doesn't run.